h1st/model/ml_model.py

```diff
@@ -40,10 +40,20 @@ def train(self, prepared_data):
            my_model_2 = MyModel()
            my_model_2.load('1st_version')
     """
+
     @property
     def base_model(self) -> Any:
         return getattr(self, "__base_model", None)
 
     @base_model.setter
     def base_model(self, value):
         setattr(self, "__base_model", value)
+
+    def predict(self, input_data: dict) -> dict:
+        """
+        Implement logic to generate prediction from data
+
+        :params data: data for prediction
+        :returns: prediction result as a dictionary
+        """
+        return self.process(input_data=input_data)
```

Comment:
Is it better to use self.process in predict function ? or just leave it empty ? 
I don't know what is the best practice, but if you look at the real example (MyMLModel), we don't implement process function. So, I feel that we don't need this line. What do you think ?

Reply:
IMO, the `process` should be implemented, and `predict` is just an alias for `process` for MLModel to provide a familiarity sense. Let me update the code.

Reply:
Shouldn't it be the other way around ? User need to implement predict() function, and process warp the predict function ? When user use my_ml_model, user can call either predict() or process(). maybe in the graph, we can call process() when we execute the graph.

Reply:
I think there will be inconsistency where users implement `process()` for non-MLModel while implementing `predict()` for MLModel. How do you think about that?